tensorflow/docs/Troubleshooting-Timeouts.md

```diff
@@ -28,13 +28,15 @@ pip install tensorflow-directml --upgrade
 
 ### #2: Ensure your GPU drivers are up-to-date
 
-We recommend using the latest stable drivers from your GPU vendor wherever possible. GPU driver updates can be downloaded from the following locations:
+When running tensorflow-directml on native Windows we recommend using the latest stable drivers from your GPU vendor wherever possible. GPU driver updates can be downloaded from the following locations:
```

Comment:
I would leave this bit off, as the Windows driver installers include both native Windows and WSL UMDs (if they don't already for all IHVs, they will at some point).

Reply:
Makes sense, removed this part in update.